config/printer-voxelab-aquila-v1.0.0-2020.cfg

```diff
@@ -0,0 +1,112 @@
+# This file contains pin mappings for the Voxelab Aquila with
+# HC32F460 on the X1_Main_Board_V1.0.0 mainboard (X1, more).
+# To use this config:
+# during: "make menuconfig" select the the HC32F460.
+# The V1.0.0 board uses 0x8000 for the Flash Application Address
```

Comment:
Not sure if these 2 directions wont be confusing, as they are not options in the make menuconfig. 

Thanks
James